zend_abstract_interface/zai_sapi/zai_sapi_ini.c

```diff
@@ -0,0 +1,43 @@
+#include "zai_sapi_ini.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+size_t zai_sapi_ini_entries_alloc(const char *src, char **dest) {
```

Comment:
Is it ok to return `0` to basically indicate an error, should we return `-1`?
Wouldn't `0` also be the result when invoked with `zai_sapi_ini_entries_alloc("", ...);` (I guess this is a valid invocation)?

Reply:
Thanks for the suggestion. I agree with you that `0` isn't always an error (as with the empty string example you provided) and it's confusing for it to represent both error and success. I'll change the return type to `ssize_t` and have `-1` represent an error. 👍

Reply:
Addressed in aa2216a.